src/app/utils/memory.ts

```diff
@@ -51,15 +52,20 @@ class MemoryManager {
 
   public async vectorSearch(contentToSearch: string) {
     const embedding = await this.generateEmbedding(contentToSearch);
+
     const result = await this.dbClient.rpc("match_documents", {
       query_embedding: embedding,
-      match_count: 1,
+      match_count: 3,
     });
 
     if (result.error) {
       console.error("ERROR: ", result.error);
     }
-    return result.data[0] || "";
+    inngest.send({
```

Comment:
**suggestion (bug_risk):** Consider handling the event sending result

The event is sent but the result isn't handled or awaited. This could lead to race conditions if other code depends on this event being processed.

```suggestion
    await inngest.send({
```